stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -603,4 +609,65 @@ end
     @test LinearAlgebra.hermitian_type(Int) == Int
 end
 
+@testset "symmetrize" begin
+    @testset "in-place" begin
+        for T in [Float32, Complex{Float32}], conj in [true, false], offset in [true, false]
+            X = rand(T, 4, 4)
+            offset && (X = OffsetArray(X, -2, -2))
+            Y = copy(X)
+            symmetrize!(X, conjugate=conj)
+            if conj
+                @test X ≈ (Y + Y') / 2
```

Comment:
Shouldn't this be `==`?   `x - x == 0` and `(x + x) / 2 == x` should always be true for finite values without overflow.